src/slopes.jl

```diff
@@ -0,0 +1,199 @@
+# Reference : Dietmar Ratz - An Optimized Interval Slope Arithmetic and its Application
+using IntervalArithmetic, ForwardDiff
+import Base: +, -, *, /, ^, sqrt, exp, log, sin, cos, tan, asin, acos, atan
+
+function slope(f::Function, x::Interval, c::Real)
+    try
```

Comment:
In case there's some elementary method that the algorithm can't build from other methods, example `sinh` (for now), then the function shouldn't return a `MethodError` but return the bound generated by automatic differentiation.